src/main/java/WordFrequencyGame.java

```diff
@@ -3,73 +3,61 @@
 import java.util.List;
 import java.util.Map;
 import java.util.StringJoiner;
-import java.io.CharArrayWriter;
-
-import java.time.LocalDateTime;
+import java.util.*;
 
 public class WordFrequencyGame {
-    public String getResult(String inputStr){
```

Comment:
better to refactor:
mysterious name: getResult -> printWordFrequency/ getWordFrequency